internal/outputer/formatter/formatter_sarif.go

```diff
@@ -0,0 +1,218 @@
+	case severity.Critical:
+		return "error"
+	case severity.High:
+		return "error"
+	case severity.Medium:
+		return "warning"
+	case severity.Low:
+		return "note"
+	default:
+		return "none"
+	}
+}
+
+func sarifProblemSeverity(s severity.Severity) string {
+	switch s {
+	case severity.Critical:
+		return "error"
+	case severity.High:
+		return "error"
+	case severity.Medium:
+		return "warning"
+	case severity.Low:
+		return "recommendation"
+	default:
+		return "recommendation"
+	}
+}
+
+func sarifSecuritySeverity(s severity.Severity) string {
+	switch s {
+	case severity.Critical:
+		return "9.0"
+	case severity.High:
+		return "7.0"
+	case severity.Medium:
+		return "4.0"
+	case severity.Low:
+		return "1.0"
+	default:
+		return "1.0"
+	}
+}
+
+func getPlaintextPolicySummary(output *scheme.Flattened, policyName string) string {
+	sFormatter := newSarifFormatter()
+	typedFormatter := sFormatter.(*sarifFormatter)
+	pf := newSarifPolicyFormatter()
+	pc := newPoliciesContent(pf, typedFormatter.colorizer)
+	return string(pc.FormatPolicy(output, policyName))
+}
+
+func getMarkdownPolicySummary(output *scheme.Flattened, policyName string) string {
+	mdFormatter := newMarkdownFormatter()
+	typedFormatter := mdFormatter.(*markdownFormatter)
+	pf := newMarkdownPolicyFormatter()
+	pc := newPoliciesContent(pf, typedFormatter.colorizer)
+	return string(pc.FormatPolicy(output, policyName))
+}
+
+type sarifColorizer struct {
+}
+
+func (sc sarifColorizer) colorize(tColor themeColor, text interface{}) string {
+	return fmt.Sprintf("%v", text)
+}
+
+// plaintext policy formatting
+type sarifPolicyFormatter struct {
+	colorizer sarifColorizer
+}
+
+func newSarifPolicyFormatter() sarifPolicyFormatter {
+	return sarifPolicyFormatter{colorizer: sarifColorizer{}}
+}
+
+func (sp sarifPolicyFormatter) FormatTitle(title string, severity severity.Severity) string {
+	color := severityToThemeColor(severity)
+	title = sp.colorizer.colorize(color, title)
+
+	return fmt.Sprintf("%s", title)
+}
+
+func (sp sarifPolicyFormatter) FormatSubtitle(title string) string {
+	return fmt.Sprintf("%s", title)
+}
+
+func (sp sarifPolicyFormatter) FormatText(depth int, format string, args ...interface{}) string {
+	return indentMultilineSpecial(depth, fmt.Sprintf(format, args...), sp.Indent(1), sp.Linebreak())
+}
+
+func (sp sarifPolicyFormatter) FormatList(depth int, title string, list []string, ordered bool) string {
+	if len(list) == 0 {
+		return ""
+	}
+
+	var sb strings.Builder
+	bullet := "*"
+	sb.WriteString(sp.FormatText(depth, "%s\n", title))
+	for i, step := range list {
+		if ordered {
+			bullet = fmt.Sprintf("%d.", i+1)
+		}
+		sb.WriteString(sp.FormatText(depth, "%s %s\n", bullet, step))
+	}
+
+	return sb.String()
+}
+
+func (sp sarifPolicyFormatter) Linebreak() string {
+	return "  \n"
+}
+
+func (sp sarifPolicyFormatter) Separator() string {
+	return "---"
+}
+
+func (sp sarifPolicyFormatter) Indent(depth int) string {
+	return strings.Repeat(" ", depth)
+}
```

Comment:
IIUC, we use that formatter for the human-readable help text.
This code seems to output a markdown-styled text instead.
So I think we can go with either:
1. Change this code to output a plaintext version.
2. Use the HumanFormatter for that. The problem with that approach is that the human formatter is currently colorful - so we'd need to set the noop colorizer to it, i.e. the sarif colorizer (can be done either by adding another new() func to the human formatter that accepts a colorizer or manually by overriding it after the creation).